### DISCUSSION TITLE
<I don't have collage ID card>

### DISCUSSION THREAD

**Discussion by MDRAHADUZZAMANLIMON** [community/community#39844] [GitHub Education]
### Select Topic Area

Question

### Body

I need to active my student pack. But our collage didn't give us our ID card. I am submitting my SSC marksheet. It isn't working?? Can you please help me??

**Comment by david-engelmann** [comment]
I would submit your problem [detailing your lack of student ID here](https://support.github.com/contact/education). You can also check out the ["What proof can I provide to support my application?" section on this page](https://support.github.com/request/education)!

**Comment by samus-aran** [comment]
Hi @MDRAHADUZZAMANLIMON 👋 

We'd love to help!  Do you have a screenshot of the rejection reasons? This might help someone give you a nudge in the right direction.

Ensure you select the most appropriate **evidence type** in the dropdown. The order of options indicates the preference of evidence with Academic Email being at the top. If you don't have an Academic email or Student ID, do try and acquire evidence that is positioned higher rather than lower in that option list.

Let us know how you get on 👍
